include/ex_actor/internal/actor_registry.h

```diff
@@ -395,4 +383,33 @@ internal::reflect::AwaitableOf<std::optional<ActorRef<UserClass>>> auto GetActor
   return internal::GetGlobalDefaultRegistry().GetActorRefByName<UserClass>(node_id, name);
 }
 
+/**
+ * @brief Configure the logging of ex_actor. Not thread-safe, please call it only when no logs are printing.
```

Comment:
`Call it only when no logs are printing. `
This is not good; normally, users are unaware when you are printing. And the behavior of breaking the rule is unspecified. It would be good not to enforce linearization, but commit to eventual consistency. It suits most cases.

Reply:
We should use something like hazard pointer. Let's do this in a separate PR.